include/cantera/thermo/CoverageDependentSurfPhase.h

```diff
@@ -0,0 +1,476 @@
+                                      AnyMap& speciesNode) const;
+
+    //! @name Methods calculating reference state thermodynamic properties
+    //! Reference state properties are evaluated at \f$ T \text{ and }
+    //! \theta^{ref} \f$. With coverage fixed at a reference value,
+    //! reference state properties are effectively only dependent on temperature.
+    //! @{
+    virtual void getEnthalpy_RT_ref(double* hrt) const;
+    virtual void getEntropy_R_ref(double* sr) const;
+    virtual void getCp_R_ref(double* cpr) const;
+    virtual void getGibbs_RT_ref(double* grt) const;
+    //! @}
+
+    //! @name Methods calculating standard state thermodynamic properties
+    //! Standard state properties are evaluated at \f$ T \text{ and } \theta \f$,
+    //! and thus are dependent both on temperature and coverage.
+    //! @{
+
+    //! Get the nondimensionalized standard state enthalpy vector.
+    /*!
+     * \f[
+     *      \frac{h^o_k(T,\theta)}{RT}
+     *          = \frac{h^{ref}_k(T) + h^{cov}_k(T,\theta)
+     *            + \int_{298}^{T} cp^{cov}_k(T,\theta)dT}{RT}
+     * \f]
+     */
+    virtual void getEnthalpy_RT(double* hrt) const;
+
+    //! Get the nondimensionalized standard state entropy vector.
+    /*!
+     * \f[
+     *      \frac{s^o_k(T,\theta)}{R}
+     *          = \frac{s^{ref}_k(T) + s^{cov}_k(T,\theta)
+     *            + \int_{298}^{T}\frac{cp^{cov}_k(T,\theta)}{T}dT}{R}
+     *            - ln\left(\frac{1}{\theta_{ref}}\right)
+     * \f]
+     */
+    virtual void getEntropy_R(double* sr) const;
+
+    //! Get the nondimensionalized standard state heat capacity vector.
+    /*!
+     * \f[
+     *      \frac{cp^o_k(T,\theta)}{RT}
+     *          = \frac{cp^{ref}_k(T) + cp^{cov}_k(T,\theta)}{RT}
+     * \f]
+     */
+    virtual void getCp_R(double* cpr) const;
+
+    //! Get the nondimensionalized standard state gibbs free energy vector.
+    /*!
+     * \f[
+     *      \frac{g^o_k(T,\theta)}{RT}
+     *          = \frac{h^o_k(T,\theta)}{RT} + \frac{s^o_k(T,\theta)}{R}
+     * \f]
+     */
+    virtual void getGibbs_RT(double* grt) const;
+
+    //! Get the standard state gibbs free energy vector. Units: J/kmol.
+    /*!
+     * \f[
+     *      g^o_k(T,\theta) = h^o_k(T,\theta) + Ts^o_k(T,\theta)
+     * \f]
+     */
+    virtual void getPureGibbs(double* g) const;
+
+    //! Get the standard state chemical potential vector. Units: J/kmol.
+    /*!
+     * \f[
+     *      \mu^o_k(T,\theta) = h^o_k(T,\theta) + Ts^o_k(T,\theta)
+     * \f]
+     */
+    virtual void getStandardChemPotentials(double* mu0) const;
+    //! @}
+
+    //! @name Methods calculating partial molar thermodynamic properties
+    //! Partial molar properties are evaluated at \f$ T \text{ and } \theta \f$,
+    //! and thus are dependent both on temperature and coverage.
+    //! @{
+
+    //! Get the partial molar enthalpy vector. Units: J/kmol.
+    /*!
+     * \f[
+     *      \overline{h}_k(T,\theta) = h^o_k(T,\theta)
```

Comment:
What does the overline mean? I don't think we use that notation elsewhere for partial molar quantities.